nifi-commons/nifi-status-history/src/main/java/org/apache/nifi/controller/status/history/StatusHistoryDumpDateRange.java

```diff
@@ -14,33 +14,36 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.apache.nifi.controller.status.history;
 
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.ZoneId;
 import java.util.Date;
 
-public class JsonNodeStatusHistoryDumpFactory implements StatusHistoryDumpFactory {
+public class StatusHistoryDumpDateRange {
```

Comment:
Minor: this could have a more general name, as it is not necessarily bounded to status history: more like a generic "number of days back interval"